Packs/Core/Integrations/CortexPlatformCore/CortexPlatformCore.py

```diff
@@ -3051,6 +3103,215 @@ def core_list_endpoints_command(client: Client, args: dict) -> CommandResults:
+    """
+    exception_rule_type = args.get("type")
+    sort_field = args.get("sort_field", "MODIFICATION_TIME")
+    sort_order = args.get("sort_order", "DESC")
+
+    default_limit = arg_to_number(args.get("limit")) or MAX_GET_EXCEPTION_RULES_LIMIT
+    page_number = arg_to_number(args.get("page", 0)) or 0
+    offset = page_number * default_limit if args.get("page") else 0
+    retrieve_all = argToBoolean(args.get("retrieve_all", False))
+
+    base_limit = MAX_GET_EXCEPTION_RULES_LIMIT if retrieve_all else default_limit
+
+    exception_rule_filter = build_exception_rules_filter(args)
+
+    if exception_rule_type in EXCEPTION_RULES_TYPE_TO_TABLE_MAPPING:
+        table_names = [EXCEPTION_RULES_TYPE_TO_TABLE_MAPPING.get(exception_rule_type)]
+    else:
+        table_names = [LEGACY_AGENT_EXCEPTIONS_TABLE, DISABLE_PREVENTION_RULES_TABLE]
+
+    all_outputs = []
+    all_raw_responses = []
+    readable_output_lines = []
+
+    for table_name in table_names:
+        records, raw_responses = get_webapp_data(
+            client=client,
+            table_name=str(table_name),
+            filter_dict=exception_rule_filter,
+            sort_field=sort_field,
+            sort_order=sort_order,
+            retrieve_all=retrieve_all,
+            base_limit=base_limit,
+            max_limit=MAX_GET_EXCEPTION_RULES_LIMIT,
+            offset=offset,
+        )
+
+        all_raw_responses.extend(raw_responses)
+
+        if records:
+            view_def = client.get_webapp_view_def({"table_name": table_name})
+            hr_output = postprocess_exception_rules_response(view_def, records)
+            all_outputs.extend(records)
+            readable_output_lines.append(hr_output)
+        else:
+            readable_output_lines.append(f"No data found for {table_name} matching the filter.")
+
+    final_readable_output = "\n".join(readable_output_lines)
+
+    return CommandResults(
+        readable_output=final_readable_output,
+        outputs_prefix=f"{INTEGRATION_CONTEXT_BRAND}.ExceptionRules",
+        outputs_key_field="ID",
+        outputs=all_outputs,
+        raw_response=all_raw_responses,
+    )
+
+
+def list_system_users_command(client, args):
+    """
+    Retrieves system user optionally filtered by email using the public api ep /public_api/v1/rbac/get_users
+    This function calls the client to fetch all available system users. If specific
+    emails are provided via the 'email' argument, it filters the results. If no
+    emails are provided, it limits the results to 50.
+    """
+    emails = argToList(args.get("email", ""))
+    if len(emails) > MAX_GET_SYSTEM_USERS_LIMIT:
+        raise DemistoException("The maximum number of emails allowed is 50.")
+    response = client.get_users()
+    data = response.get("reply", {})
+    if emails:
+        data = [user for user in data if user.get("user_email") in emails]
+    if len(data) > MAX_GET_SYSTEM_USERS_LIMIT:
+        data = data[:MAX_GET_SYSTEM_USERS_LIMIT]
```

Comment:
Why should we limit the response if limit was not provided?
How can we get all the users without pagination?